src/main/java/com/marklogic/kafka/connect/source/MarkLogicSourceConfig.java

```diff
@@ -0,0 +1,87 @@
+package com.marklogic.kafka.connect.source;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigDef.Importance;
+import org.apache.kafka.common.config.ConfigDef.Type;
+
+import java.util.Map;
+
+/**
+ * Defines configuration properties for the MarkLogic source connector.
+ */
+public class MarkLogicSourceConfig extends AbstractConfig {
+
+    public static final String CONNECTION_HOST = "ml.connection.host";
```

Comment:
Doesn't need to happen yet, but I think we'll want a base class of `MarkLogicConfig` with the common properties (probably just connection-related stuff) which extends `AbstractConfig`.

Reply:
I'll do this for the next PR.